i18n/zh/docusaurus-theme-classic/footer.json

```
@@ -23,6 +23,10 @@
     "message": "博客",
     "description": "The label of footer link with label=Blog linking to blog"
   },
+  "link.item.label.Adopters": {
+    "message": "案例",
```

```suggestion
    "message": "采用者",
```
？ 将来可能还需要加入 `Case Study` ,届时可能会用到 `案例` 一词。

---

agree with u suggestion. and I was updated.

---

我看到预览版本已经改过了，这里`案例`两字不用改吗？

---

footer 加在页面底部，抱歉，遗漏了一并修改啦.

---

No worries.